R/ssh_connector.R

```diff
@@ -142,7 +142,7 @@ ssh_connector <- function(data = teal.data::teal_data(),
           within(
             data,
             {
-              ssh_args <- ssh_authenticator(host = host)
+              ssh_args <- teal.connector.ssh::ssh_authenticator(host = host)
```

Comment:
I think this triggers R CMD CHECK Warning 
https://github.com/insightsengineering/teal.connectors.ssh/actions/runs/10215201723/job/28264147809?pr=8#step:41:108
```r
* checking dependencies in R code ... WARNING
'::' or ':::' import not declared from: ‘teal.connector.ssh’
```

Reply:
should we attach `teal.connector.ssh` (with `library`) in `data = teal_data()` to get rid of this warning?

Reply:
That was a typo I made when committing :facepalm: sorry.

I got complacent as I expected a URL failing and didn't notice the R CMD check failing.

It's now corrected and I've also updated all the URLs (previously we were expecting the release, so some were pointing to `latest-tag`).

All green now